suites/test_module_content.py

```diff
@@ -139,3 +146,23 @@ def verify_module_content(self, api_auth, setup_test_products):
                      data_from_published_module.json()["module"]["isPartOf"][i]["pantheon_env"], equal_to(env))
       print(*relative_url1)
       check_that("Relative url to", relative_url1, has_item("/"+published_assembly_relative_url))
+      time.sleep(10)
+      # Test to verify external proxy url, pantheon URL and CP url resolve image assets correctly
+      proxies = {
+          "http": proxy_server,
+          "https": proxy_server,
+      }
+      body = data_from_published_module.json()["module"]["body"]
```

Comment:
@noopurmath3 please add the test check in the test description also `@lcc.test`.
thanks.